packages/beacon-node/src/chain/chain.ts

```diff
@@ -222,13 +222,13 @@ export class BeaconChain implements IBeaconChain {
         ? anchorState
         : createCachedBeaconState(anchorState, {
             config,
-            pubkey2index: new PubkeyIndexMap(),
-            index2pubkey: [],
+            finalizedPubkey2index: new PubkeyIndexMap(),
+            finalizedIndex2pubkey: [],
           });
 
     // Persist single global instance of state caches
-    this.pubkey2index = cachedState.epochCtx.pubkey2index;
-    this.index2pubkey = cachedState.epochCtx.index2pubkey;
+    this.pubkey2index = cachedState.epochCtx.finalizedPubkey2index; // TODO: Double check if these two are correct
```

Comment:
global instaces should/would be tracking the finalized maps , so that would be correct